backend/src/main/java/vn/edu/hust/nmcnpm_20242_n3/dto/StaffDTO.java

```diff
@@ -0,0 +1,94 @@
+package vn.edu.hust.nmcnpm_20242_n3.dto;
+
+import java.util.Date;
+
+public class StaffDTO {
+    private String id;
+    private String name;
+    private String userName;
+    private String email;
+    private String password;
+    private Date createdAt;
+    private Date updatedAt;
+    private RoleDTO role;
```

Comment:
I'm not sure whether or not we will the the whole role to be returned to the user. For now, should we just stick to using `role` as an `Enum`?